src/transaction_builder.h

```diff
@@ -262,7 +261,7 @@ class TransactionBuilder
     const CCoinsViewCache* coinsView;
     CCriticalSection* cs_coinsView;
     CMutableTransaction mtx;
-    CAmount fee = DEFAULT_FEE;
+    CAmount fee = LEGACY_DEFAULT_FEE;
```

Comment:
We should figure out whether there's a reason to initialize this to `LEGACY_DEFAULT_FEE` or whether it should now either not have a default or have a default value of zero. 1000 ZAT no longer makes much sense.

Reply:
Yup. I think it makes more sense to change this in another PR that is focussed on trying to reduce the use of `LEGACY_DEFAULT_FEE`.